core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala

```diff
@@ -789,6 +793,56 @@ class EventsByTagStrictBySeqNoEarlyFirstOffsetSpec
   }
 }
 
+class EventsByTagLongRefreshIntervalSpec
```

Comment:
This wasn't caught in the previous test as we set the refresh interval very low to speed up tests